infra/integration-image/scripts/ubuntu-remove-intermediate-ca.sh

```diff
@@ -1,4 +1,18 @@
 #!/usr/bin/env bash
+# Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#!/usr/bin/env bash
```

Comment:
Hm... well that means the header check script still has some bugs when it comes to the `fix` option; I used that to insert this header.